certified-connectors/PDF4me Connect/apiDefinition.swagger.json

```diff
@@ -1330,17 +1306,62 @@
                   ],
                   "type": "object"
                 },
+                "text": {
+                  "description": "Stamp text",
+                  "title": "Text",
+                  "type": "string",
+                  "x-ms-visibility": "important"
+                },
                 "pages": {
                   "description": "Mention the page numbers like 1,2",
                   "title": "Pages",
                   "type": "string",
+                  "default": "all",
                   "x-ms-visibility": "important"
                 },
-                "text": {
-                  "description": "Stamp text",
-                  "title": "Text",
+                "alignX": {
+                  "default": "left",
+                  "description": "Set alignment in horizontal scale",
+                  "enum": [
+                    "left",
+                    "center",
+                    "right"
+                  ],
+                  "title": "Horizontal Alignment",
                   "type": "string",
                   "x-ms-visibility": "important"
+                },
+                "alignY": {
+                  "default": "top",
+                  "description": "Set alignment in vertical scale",
+                  "enum": [
+                    "top",
+                    "middle",
+                    "bottom"
+                  ],
+                  "title": "Vertical Alignment",
+                  "type": "string",
+                  "x-ms-visibility": "important"
+                },
+                "marginXInMM": {
+                  "description": "Margin from Left or Right origin of the image in MilliMeter",
+                  "default": "0",
+                  "title": "Horizontal Margin in mm.",
+                  "type": "string",
+                  "x-ms-visibility": "important"
+                },
+                "marginYInMM": {
+                  "description": "Margin from Top or Bottom origin of the image in MilliMeter",
+                  "default": "0",
+                  "title": "Vertical Margin in mm.",
+                  "type": "string",
+                  "x-ms-visibility": "important"
+                },
+                "opacity": {
+                  "description": "Give number between 0 to 100. 0 is completely transparent.                100 for complete visible.",
```

Comment:
@varunynoox Please remove the extra spaces in the description value.

Reply:
extra space has been removed